docs/developer/metadata.mdx

```diff
@@ -108,6 +122,44 @@ Response:
 }
 ```
 
+## Selecting fields to return
+
+By default, `metafields` (`privateMetafields`) will return all metadata that the object has assigned. You can use the optional `keys` argument to select the fields you want to return. We highly recommend that you only fetch the fields your application understands as it limits the amount of information being needlessly transferred.
```

Comment:
```suggestion
By default, `metafields` (`privateMetafields`) will return all metadata assigned to an object. You can use the optional `keys` argument to select the fields you want to return. We highly recommend that you only fetch the fields your application understands as it limits the amount of information being needlessly transferred.
```

@patrys What do you mean by "fields your application understands"?

Reply:
Maybe let's change to `uses` or `needs`? (its also not only data but interface segregation principle, not relying on fields that are not needed)